### PR TITLE
ExoMedia cleanup

### PR DESCRIPTION
**Pull request by zsmb13**
### 🎯 Goal

Remove the code of an old version of the [ExoMedia library](https://github.com/brianwernick/ExoMedia) from our project, which was directly copy-pasted there instead of being added as a dependency.

### 🛠 Implementation details

Removes all ExoMedia code from the project, as well as the direct dependency on ExoPlayer which is no longer needed. ExoMedia is now pulled in as a Jitpack dependency instead (while it's not published to MavenCentral).

### 🎨 UI Changes

https://user-images.githubusercontent.com/12054216/117192943-96412d00-ade2-11eb-9bdc-4f85c9b2ad65.mp4

### 🧪 Testing

Send video, tap the sent attachment to play it with the new player.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes
- [ ] ~New code is covered by unit tests~
- [ ] ~Comparison screenshots added for visual changes~
- [ ] ~Affected documentation updated (CMS, cookbooks, tutorial)~
- [x] Reviewers added

### CODE REVIEW

File: buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt [hunk 1]
```diff
@@ -30,7 +30,6 @@ private const val DRAWABLETOOLBOX_VERSION = "1.0.7"
 private const val DOKKA_VERSION = "1.4.30"
 private const val DOTENV_VERSION = "5.2.2"
 private const val ESPRESSO_VERSION = "3.3.0"
-private const val EXOPLAYER_VERSION = "2.12.1"
```
Comment (zsmb13) [inline buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt:33]: We didn't use ExoPlayer directly anywhere, we're using it via the ExoMedia wrapper

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/AttachmentMediaActivity.java [hunk 2]
```diff
@@ -63,6 +63,7 @@ private void init() {
      * @param url media url
      */
     public void playVideo(String url) {
+        videoView.isPlaying(); // Workaround to init some internals of the library
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/AttachmentMediaActivity.java:66]: See explanation here https://github.com/brianwernick/ExoMedia/issues/732

**Review by adasiewiczr — APPROVED** [review]
LGTM!

**Review by ogkuzmin — APPROVED** [review]
Wow ❤️

**Comment by zsmb13** [comment]
Will resolve https://github.com/GetStream/stream-chat-android/issues/1843

**Pull request merged**